jest.setup.ts

```diff
@@ -1 +1,6 @@
 import '@testing-library/jest-dom/extend-expect';
+import 'jest-extended/all';
+import { configure } from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
+
+configure({ adapter: new Adapter() });
```

Comment:
Can we write tests using `react-testing-library`, e.g. [Card tests](https://github.com/zer0-os/zUI/blob/master/src/components/Card/Card.test.tsx), [Button tests](https://github.com/zer0-os/zUI/tree/master/src/components/Button)? If not we should agree on a standard so we're not doing half-enzyme-half-RTL.

Reply:
I can't seem to figure out how to do shallow rendering in RTL. Maybe you can show me an example? The Button tests seem to mock the sub component but still perform a full render on the mock.

Reply:
We aren't shallow rendering in our tests - I generally prefer to test things as close as possible to how a user will use them. I'm definitely open to the idea of using shallow rendering though, as long as we're careful about which implementation details we're testing (if any).

I think I chatted to Joel about this a while ago, maybe we need to start the conversation again! If you're happy to use RTL for now we can push on and chat async, otherwise we can expedite the conversation.